### ISSUE TITLE
The infamous CLA check is back

### ISSUE THREAD

**Issue by ederag** [projecthamster/hamster#589]
@elbenfreund, it's [back](https://github.com/projecthamster/hamster-shell-extension/issues/135#issuecomment-309821921)
![Screenshot_20200320_234148](https://user-images.githubusercontent.com/10962809/77211477-619de380-6b04-11ea-9cf1-a536bab5b956.png)

It's most probably a mistake, as you well know this code is pure GPL,
and can *never* be re-licensed otherwise.

Please remove it, that scares contributors off,
and increases the risk of *someone else* putting a CLA on this by *mistake*.

**Comment by ederag** [comment]
After 859a89e51ac003377ddfe0d6f360c92462acff81, the check disappeared from #573.
But it is still there for the other PR.

**Comment by matthijskooijman** [comment]
Looking at recent pull requests, I see no CLA checks anymore, so I think this is now properly disabled. I'll close this, we can reopen when this turns out to be false.

**Comment by matthijskooijman** [comment]
I'd rather just delete the comments, let me do that.

**Comment by matthijskooijman** [comment]
Seems there was just one CLA comment left, I deleted it. I also deleted @ederag's followup comments since they referred already deleted CLA comments, which probably only adds confusion.

**Comment by ederag** [comment]
The checks still appear. 
![Screenshot_20200428_153039](https://user-images.githubusercontent.com/10962809/80493087-39a46a00-8965-11ea-858f-9bb2f11a5792.png)

Don't you dare delete any of my comments again without prior discussion.
That's disrespectful.

**Comment by matthijskooijman** [comment]
Ah, I was looking at the CLA assistant comments, but there are indeed still (or again) pending checks. I see this is only the case on some PRs and not new ones, so that probably means that the checks are properly disabled now, but the existing ones still linger. Not sure if they can be removed, maybe @ederag can.

As for deleting comments, sorry for that. It seemed better to delete the CLA comments, and with that also your comment since it would otherwise not make sense. When I found no further CLA comments, I thought you were being overzealous in adding comments everywhere, but it seems I just missed the pending check.

In any case, I reposted some comments, and posted one additional one, so I think all pending checks have not been covered and this can be closed again. Agreed?
